src/osemgrep/cli_scan/Rule_fetching.ml

```diff
@@ -40,8 +41,7 @@ type rules_and_origin = {
 }
 
 (* TODO? more complex origin? Remote of Uri.t | Local of filename | Inline? *)
-and origin = Common.filename option (* None for remote files *)
-[@@deriving show]
+and origin = Fpath.t option (* None for remote files *) [@@deriving show]
```

Comment:
So deriving show is working on Fpath.t too?

Reply:
I guess only `Fpath.pp` needs to be available, not `Fpath.show`.